frontend/src/dummyData/statuses.ts

```diff
@@ -8,14 +8,14 @@ const mastodonRawStatuses: MastodonStatus[] = [
 		`
 		<p>Fine. I'll use Wildebeest!</p>
 		<p>It does look interesting:
-			<a href="https://blog.cloudflare.com/welcome-to-wildebeest-the-fediverse-on-cloudflare/"
+			 <a href="https://blog.cloudflare.com/welcome-to-wildebeest-the-fediverse-on-cloudflare/"
```

Comment:
??

Reply:
😅 

yeah I am removing all tabs and newlines, and I did want a space between `interesting:` and the link

would the space on line 10 be better? and/or also moving the `<a` there?

Reply:
Hmm. So in that case I think you should be replacing the whitespace below with a space, not an empty string.

Reply:
Or if you really want to do it here (and maintain the current replacement behaviour) then I think you should use an HTML encoding: `&#32`.

Reply:
> Hmm. So in that case I think you should be replacing the whitespace below with a space, not an empty string.
the issue is that I am clearing up all the tabs and newlines, it's not like I am contracting them or the like, so I don't think I could change the replacement to keep a tab (unless I don't replace the specific tab found after `interesting:` but that doesn't really look clean 😕)

Reply:
> Or if you really want to do it here (and maintain the current replacement behaviour) then I think you should use an HTML encoding: &#32.
cool thanks I'll use that 😃

Reply:
> the issue is that I am clearing up all the tabs and newlines, it's not like I am contracting them or the like, so I don't think I could change the replacement to keep a tab (unless I don't replace the specific tab found after interesting: but that doesn't really look clean 😕)

Not sure if we are on the same page here. I was suggesting that you replace all the tabs and newlines with a space rather than an empty string.  E.g. `.replace(/[\t\n]/g, ' ')`. I think this should have the same effect but not cause text span to run into each other, right?

Reply:
I see 🙂 

Anyways that wouldn't work as we have multiple tab spaces, so I tried `.replace(/[\t\n]+/g, ' ')` it still produces a bit more spaces than necessary, so I tried concatenating `.replace(/\s+/g, ' ')` to squash those but then I realized that we could have spaces before and after tags which aren't really consecutive but produce a result with too many spaces anyways, since this is only for a mock status I actually ended up just using `&#32` as you suggested

if you do prefer not to have it and use replacements alone I can also use the above ones and have a small number of extra spaces, either way this is just to show our mock content so I don't think it is too important, nor it needs to be precise

Let me know if `&#32` is ok or if you'd prefer a different solution 🙂